ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -39,6 +39,26 @@ class InterceptingSpec: QuickSpec {
 				expect(object.counter) == 2
 			}
 
+			it("should send a value when the selector is invoked without implementation") {
+				let selector = #selector(TestProtocol.optionalMethod)
+
+				let signal = object.reactive.trigger(for: selector,
+				                                     from: TestProtocol.self)
+				expect(object.responds(to: selector)) == true
+
+				var counter = 0
+				signal.observeValues { counter += 1 }
+
+				expect(counter) == 0
+
```

Comment:
Not if you specify a protocol. Then in `NSObject._rac_setupInvocationObservationForSelector`, it would look if the selector exists in the protocol and create an implementation for the selector (`_objc_msgForward `) correspondingly.